packages/app-ui/src/components/atoms/GenericTokenIcon/GenericTokenIcon.tsx

```
@@ -3,6 +3,8 @@
 import { type FC } from 'react'
 import loadable from '@loadable/component'
 
+import { safeBTOA } from '../../../../../app-utils/src'
```

ESLint

[error] import statements should have an absolute path

(no-relative-import-paths/no-relative-import-paths)